### ISSUE TITLE
subscribeInvoice type incorrect, returns Readable not Promise

### ISSUE THREAD

**Issue by wbobeirne** [RadarTech/lnrpc#36]
This is the `subscribeInvoice` type currently provided:

```ts
LnRpc.subscribeInvoices(args?: InvoiceSubscription | undefined): Promise<Readable<Invoice>>
```

However the following code:

```ts
node.subscribeInvoices().then(stream => {
  stream.on('data', chunk => {
    console.log(chunk);
  });
});
```

results in the following error:

```
UnhandledPromiseRejectionWarning: TypeError: node_1.node.subscribeInvoices(...).then is not a function
```

Doing a quick log of the call:

```ts
const stream = node.subscribeInvoices();
console.log(stream);
```

Prints this, indicating it returns a readable, no promise wrap:

```
ClientReadableStream {
  _readableState:
    ReadableState {
      ...
```

I'm not sure if it's just the type that's wrong, or the non-promise behavior that's wrong, but I currently have to work around this by doing:

```ts
import { Invoice, Readable } from '@radar/lnrpc';

const stream = node.subscribeInvoices() as any as Readable<Invoice>;
stream.on('data', chunk => {
  console.log(chunk);
});
```

**Comment by cavanmflynn** [comment]
Ah, you're correct. Our usage of async/await hid the issue. Patching now.

**Comment by cavanmflynn** [comment]
Published in [0.6.0-beta.1](https://www.npmjs.com/package/@radar/lnrpc/v/0.6.0-beta.1)
